Makefile

```diff
@@ -474,12 +465,14 @@ endif
 	@ecs-cli compose -f docker-compose.prod.yml up
 
 
-
-# Deprecated targets
 deploy:
-	@echo $@ ERROR: see deployment steps for accelerate.
-	@echo see: https://github.com/masschallenge/standards/blob/376d290b41a202acc5c2263d7275ba4a57330ad7/create_new_release.md#deploy-to-staging
+	@if [ "$$IMAGE_TAG" != "development" ] && [ "$$IMAGE_TAG" != "AC-6678" ]; then exit 1; fi;
```

Comment:
is this hard-coded branch name needed once merged?

Reply:
the hard-coded `AC-6678` can go..and there's a [ticket](https://masschallenge.atlassian.net/browse/AC-6754) to clean this up once merged.